vitest.extend.ts

```diff
@@ -15,6 +32,20 @@ export const roundSnapshotOptions = {
 };
 
 expect.extend({
+  toEqualSVG(actual: string, expected: string) {
```

Comment:
added this matcher so it is equivalent to the custom one that exists in qunit

Reply:
Yeah for this one in the gradient PR i m doing i had to find a workaround, but maybe this is nicer.